ADOPTERS.md

```diff
@@ -14,6 +14,7 @@
 | End-user | CloudRaft| 2024 | [link](https://cloudraft.io/) | CloudRaft is using KubeVirt to build a GPU Cloud platform for AI workload. KubeVirt allows end users such as Data scientists to experiment on powerful remote VMs equipped with GPUs and AI/ML packages.  |
 | End-user | CoreWeave | 2020 | [link](https://www.coreweave.com) | A Kubernetes native cloud provider with focus on GPUs at scale. KubeVirt allows us to co-locate non-containerizable workloads such as Virtual Desktops next to compute intensive containers executing on bare metal. All orchestrated via the Kubernetes API leveraging the same network policies and persistent volumes for both VM and containerized workloads. |
 | End-user | Genesis Cloud | 2022 | [link](https://genesiscloud.com/) | Genesis Cloud is basing its public cloud offering for instances with GPUs and other accelerators on kubevirt. |
+| End-user | Habana Labs | 2024 | [link](https://habana.ai/) | Habana Labs utilizes KubeVirt and builds upon open-source projects such as Kubernetes and KubeVirt within its core data centers to empower products like  Habana Labs uses KubeVirt and built on open-source projects like [Intel® Gaudi® 3](https://habana.ai/products/gaudi3/) |
```

Comment:
The wording looks a little bit redundant. Can you Try to rephrase it a little and maybe include if you are using it to test/build/certify etc your porducts?

Reply:
Hi @fabiand I cleaned the duplication I had. please check again. we use Kubevirt for AI workloads based on the Intel Gaudi AI accelerator in our internal DC

Reply:
Awesome, thanks, looks good!

FWIW I think understanding the reasoning why running it in VMs vs plain pods is something that many people might be interested in - aka session content this seems to be :)

Reply:
Please change "Habana Labs" to "Intel Gaudi" Thanks